lib/common/rules.c

```diff
@@ -367,8 +431,13 @@ pe__eval_date_expr(const xmlNode *expr, const crm_time_t *now,
             crm_time_add_seconds(start, 1);
             pcmk__set_time_if_earlier(next_change, start);
         }
+        crm_time_free(start);
 
     } else if (pcmk__str_eq(op, PCMK_VALUE_LT, pcmk__str_casei)) {
+        crm_time_t *end = NULL;
+
+        pcmk__xe_get_datetime(expr, PCMK_XA_END, &end);
```

Comment:
I missed that one :)

It's only possible with schema validation disabled. I've avoided changing behavior even in that situation, but I think I'll let this one slide -- it would be a creative violation of the documented syntax, and supporting it would be really ugly.